packages/ui/src/common/components/Activities/ActivityContent.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+import { ApplicationWithdrawnContent } from '@/working-groups/components/Activities/ApplicationWithdrawnContent'
+import { AppliedOnOpeningContent } from '@/working-groups/components/Activities/AppliedOnOpeningContent'
+import { BudgetSetContent } from '@/working-groups/components/Activities/BudgetSetContent'
+import { BudgetSpendingContent } from '@/working-groups/components/Activities/BudgetSpendingContent'
+
+import { Activity } from '../../types'
+
+interface Props {
+  activity: Activity
+}
+
+export const ActivityContent = React.memo(({ activity }: Props) => {
+  switch (activity.eventType) {
+    case 'AppliedOnOpeningEvent':
+      return <AppliedOnOpeningContent activity={activity} />
+    case 'BudgetSpendingEvent':
+      return <BudgetSpendingContent activity={activity} />
+    case 'ApplicationWithdrawnEvent':
+      return <ApplicationWithdrawnContent activity={activity} />
+    case 'BudgetSetEvent':
+      return <BudgetSetContent activity={activity} />
+    default:
+      return <div />
+  }
```

Comment:
Not great not terrible ;) Let's refactor that later when we have more activities from other modules.